src/main/java/org/checkerframework/flexeme/Util.java

```diff
@@ -53,9 +55,10 @@ private static <V> Set<V> mergeSets(final Set<V> left, final Set<V> right) {
     /**
      * Create a new mutable set containing one element. This is a helper method to avoid having to write
      * {@code new HashSet<>(Collections.singleton(value))} everywhere.
+     *
      * @param value the value to put in the set
+     * @param <V>   the type of the set elements
```

Comment:
Changes like this are good.  Could you make them in a separate pull request (no need to remove them here), so they can be quickly reviewed?  When main is then pulled into this branch, its diffs will be smaller.